docs/source/gbd2019_models/causes/ihd/index.rst

```diff
@@ -116,331 +112,224 @@ Assumptions and Limitations
 Cause Model Diagram
 +++++++++++++++++++
 
-State and Transition Data Tables
-++++++++++++++++++++++++++++++++
-
-This section gives necessary information to software engineers for building the model. 
-This section usually contains four tables: Definitions, State Data, Transition Data and Data Sources.
-
-Definitions
+MI
 """""""""""
 
-This table contains the definitions of all the states in **cause model diagram**. 
+.. image:: cause_model_mi.svg
 
-.. list-table:: State Definitions
-   :widths: 5 5 20
-   :header-rows: 1
+Angina
+"""""""""""
 
-   * - State
-     - State Name
-     - Definition
-   * - 
-     - 
-     - 
-   * - 
-     - 
-     - 
+.. image:: cause_model_angina.svg
 
-For example, the *Definitions* table for *SIR* and *With-Condition and Free of Condition Model* models are as below:
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
 
-**SIR Model**
+Definitions
+"""""""""""
 
 .. list-table:: State Definitions
-   :widths: 5 5 20
+   :widths: 1, 10, 15
    :header-rows: 1
 
    * - State
      - State Name
      - Definition
    * - S
-     - Susceptible
-     - Susceptible to {cause name}
-   * - I
-     - Infected
-     - Infected with {cause name}
-   * - R
-     - Recovered
-     - Infected with {cause name}
-
-
-**With-Condition and Free of Condition Model**
-
-.. list-table:: State Definitions
-   :widths: 1, 5, 10
-   :header-rows: 1
-
-   * - State
-     - State Name
-     - Definition
-   * - C
-     - With **C**\ ondition
-     - Born with {cause name}
-   * - F
-     - **F**\ ree of Condition
-     - Born without {cause name}
-
-Include states, their names and definitions appropriate to your model.
+     - **S**\ usceptible
+     - Susceptible to IHD
+   * - A
+     - **A**\ cute Myocardial Infarction (MI)
+     - Simulant that experiences acute MI symptoms
+   * - P
+     - **P**\ ost-MI IHD
+     - Simulant that experiences angina and asymptomatic ischemic heart
+       disease following myocardial infarction; survival to 28 days following
+       incident MI
+   * - S2
```

Comment:
It will be confusing to the engineers (and I confess it is also confusing to me) that there are two states labeled "Susceptible to IHD".  Is it possible to me in A and A2 simultaneously? If they are mutually exclusive, perhaps they should be in the same model diagram.